static/app/views/starfish/modules/databaseModule/databaseTableView.tsx

```diff
@@ -144,7 +121,7 @@ export default function APIModuleView({
         </Hovercard>
       );
     }
-    if (column.key === 'p75') {
+    if (['p75', 'total_time'].includes(column.key)) {
```

Comment:
I think if you convert this, the error would disappear
```suggestion
    if (['p75', 'total_time'].includes(column.key.toString())) {
```